retrofit-adapters/rxjava2/pom.xml

```diff
@@ -49,5 +49,10 @@
       <artifactId>mockwebserver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.squareup.retrofit2</groupId>
+      <artifactId>retrofit-mock</artifactId>
+      <version>2.4.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
This needs to be in the "test" scope, otherwise you leak the dependency to consumers.